style.css

```diff
@@ -1,3 +1,8 @@
+body {
+    background: white; /* #dbdbdb; */
```

Comment:
I feel that the white background, in combination with the red outlines, is a bit hard on the eyes